source/css/sass/mixins/_typography.scss

```diff
@@ -68,14 +68,17 @@
 }
 
 @mixin _font-variant-position($position) {
-  font-size: 75%;
-  line-height: 0;
-  position: relative;
-  vertical-align: baseline;
```

Comment:
We need to keep this in following https://github.com/libero/pattern-library/pull/50#discussion_r253898229, relies on using `<sup>` otherwise.